1001-2000/LIT1297Dersan.xml

```diff
@@ -33,17 +35,30 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
-         <langUsage><language ident="en">English</language></langUsage>
+         <langUsage>
+           <language ident="en">English</language>
+           <language ident="gez">Gǝʿǝz</language>
+         </langUsage>
       </profileDesc>
       <revisionDesc>
          <change who="PL" when="2016-03-21"> Created file from google
             spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: text record</change>
+         <change who="MKr" when="2020-12-04">Updatek transliteration, added summary, relations and bibliography</change>
```

Comment:
Updated

Reply:
(I have preapproved you can however see if you correct the typo before merging)

Reply:
If you meant the typo in "updated, its'done.